services/core-web/src/components/mine/Permit/MinePermitInfo.tsx

```diff
@@ -431,6 +435,21 @@ export const MinePermitInfo: FC<MinePermitInfoProps> = (props) => {
             <ExplosivesPermit isPermitTab />
           </>
         </Tabs.TabPane>
+        {isFeatureEnabled(Feature.VERIFIABLE_CREDENTIALS_2) && (
+          <Tabs.TabPane
```

Comment:
I think Tabs.TabPane gives deprecation warnings- can use items instead
https://4x.ant.design/components/tabs/#header

Reply:
Agree, and yes, but I didn't want to increase scope any further as I'd have to rewrite more of this file which wasn't really the target of this work.